lumicks/pylake/kymotracker/kymoline.py

```diff
@@ -345,19 +346,44 @@ def estimate_diffusion_ols(self, max_lag=None):
 
         1) Michalet, X., & Berglund, A. J. (2012). Optimal diffusion coefficient estimation in
         single-particle tracking. Physical Review E, 85(6), 061916.
+        2) Bullerjahn, J. T., von Bülow, S., & Hummer, G. (2020). Optimal estimates of
+        self-diffusion coefficients from molecular dynamics simulations. The Journal of Chemical
+        Physics, 153(2), 024116.
 
         Parameters
         ----------
         max_lag : int (optional)
-            Number of lags to include. When omitted, the method uses an optimal number of lags
-            as determined by [1].
+            Number of lags to include. When omitted, the method will choose an appropriate number
+            of lags to use.
         """
         frame_idx, positions = np.array(self.time_idx, dtype=int), np.array(self.position)
         max_lag = max_lag if max_lag else determine_optimal_points(frame_idx, positions)[0]
         return estimate_diffusion_constant_simple(
             frame_idx, positions, self._line_time_seconds, max_lag
         )
 
+    @deprecated(
+        reason=(
+            "Diffusion constant estimation was moved to the function `estimate_diffusion` "
+            "which provides additional metadata."
```

Comment:
Maybe just some slight re-wording
```suggestion
            "This method is replaced by `Kymoline.estimate_diffusion()`  to allow more flexibility"
            "in the choice of optimization algorithms and provides additional metadata."
```

Reply:
👍